test/integration/lib/storages/RedisProxyCache.int.test.ts

```diff
@@ -41,7 +43,12 @@ const redisClusterProxyConfig = fixtures.redisClusterProxyConfigDto({
 logger.info('redis proxyConfigs', { redisClusterProxyConfig, redisProxyConfig });
 
 describe('RedisProxyCache Integration Tests -->', () => {
-  const runUseCases = (proxyCache: IProxyCache, anotherProxyCache: IProxyCache) => {
+  const runUseCases = (proxyCache: IProxyCache<RedisClientType>, anotherProxyCache: IProxyCache<RedisClientType>) => {
+    beforeEach(() => {
+      proxyCache.client.flushall();
```

Comment:
I believe, it's not _proxyCache lib_ responsibility to clean up Redis (or do whatever else, not specified by interface). 

You can create a separate redis connection to perform any Redis-related operations, e.g.:
https://github.com/mojaloop/inter-scheme-proxy-cache-lib/blob/feat/csi-15/test/unit/lib/storages/RedisProxyCache.test.ts#L40